src/atoms/InputField/InputField.css

```diff
@@ -0,0 +1,9 @@
+.input {
```

Comment:
We can use module.css here, so to avoid namespace collision, module.css will generate a unique class name even for two classes which shares same name.

Reference : https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/